src/MizarWidgets/SamplingWithFrameTrackWidget.cpp

```diff
@@ -20,6 +22,13 @@ using ::orbit_mizar_base::Comparison;
 SamplingWithFrameTrackWidget::SamplingWithFrameTrackWidget(QWidget* parent)
     : QWidget(parent), ui_(std::make_unique<Ui::SamplingWithFrameTrackWidget>()) {
   ui_->setupUi(this);
+
+  OnMultiplicityCorrectionCheckBoxClicked(Qt::CheckState::Checked);
+
+  QObject::connect(ui_->multiplicity_correction_, &QCheckBox::stateChanged, this,
+                   &SamplingWithFrameTrackWidget::OnMultiplicityCorrectionCheckBoxClicked);
```

Comment:
I orient my naming at the qt stignals. Aka, I would name the method `OnMultiplicityCorrectionCheckBoxStateChanged`, or use the signal `&QCheckBox::clicked(bool checked)`. (The latter is maybe even nicer because you get a `bool` instead of that `Qt::CheckState`)

Reply:
Great point, thank you. I've missed `clicked` signal. Probably, because it's inherited.